cds/orgsvc/C#/DiscoveryService/SampleMethods.cs

```diff
@@ -14,39 +14,39 @@ namespace PowerApps.Samples
     public partial class SampleProgram
     {
         /// <summary>
-        /// Gets organization data for user in all regions when data center is unknown
+        /// Gets organization data for user in commercial region cloud is unknown
         /// </summary>
```

Comment:
Not quite a sentence. Recommend simply "Gets organization data for user."

Reply:
Fixed